src/main/java/address/browser/BrowserManager.java

```diff
@@ -123,6 +122,12 @@ public synchronized void loadProfilePage(ReadOnlyViewablePerson person) {
         selectedPersonUsername.addListener(listener);
     }
 
+    private void configureGithubPageTasks(List<Page> pages) {
+        List<GithubProfilePage> githubProfilePages = pages.stream().map(p
+                -> new GithubProfilePage(p)).collect(Collectors.toCollection(ArrayList::new));
+        githubProfilePages.stream().forEach(GithubProfilePage::setupPageAutomation);
+    }
```

Comment:
- we can combine the two so no need for the list.
- we can use method reference for the creation of github profile page.

`pages.stream().map(GithubProfilePage::new).forEach(GithubProfilePage::setupPageAutomation);`